### PR TITLE
merge master to develop

### PR DESCRIPTION
**Pull request by MoisesFlores22**
This PR is to make develop the generic file for the chat

### CODE REVIEW

File: src/Attachment.tsx [hunk 1]
```diff
@@ -22,20 +113,21 @@ const Media = (props: {
     onLoad?: () => void,
     onClick?: () => void,
 }) => {
-    const { type, ... mediaProps } = props; // this allows us to keep 'type' out of the final HTML
-    switch (type) {
+    switch (props.type) {
         case 'video':
-            return <video controls {... mediaProps } />;
+            return <Video { ... props } />
```
Comment (HzJavier) [inline src/Attachment.tsx]: Why did we deleted the `;`?

Comment (MoisesFlores22) [inline src/Attachment.tsx]: mmm actually the deletion was made by microsoft

Comment (HzJavier) [inline src/Attachment.tsx]: Sorry, I lacked context :/

**Comment by MoisesFlores22** [comment]
@HzJavier no problem :)

File: src/Attachment.tsx (hunk 1 above)
Comment (MoisesFlores22) [inline src/Attachment.tsx]: no problem, in this webchat that is a common thing XD

**Pull request merged**